users/serializers.py

```diff
@@ -45,3 +47,74 @@ def validate(self, attrs):
         data.pop('username')
 
         return data
+
+
+class PasswordUpdateSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """Serializes old and new passwords for updating user password. """
```

Comment:
I suggest we will replace it for
`"""Serializes both the old and new passwords provided by the current user to change their password. """`